app/konnect/getting-started/configure-runtime.md

```diff
@@ -33,45 +32,40 @@ organization admin permissions by default.
 
 ## Set up a New Runtime Instance
 
-1. From the left navigation menu, open **Runtimes**.
+1. From the left navigation menu, open **Runtime Manager**.
 
-    For the first runtime, the page opens to a **Configure New Runtime** form.
+1. Select a runtime group. [to do: check what happens if you only have the default group - is it open automatically?]
 
-    Once configured, this page lists all runtimes associated with the
-    {{site.konnect_saas}} account.
+1. Click **Configure Runtime**.
 
-2. Click **Copy Script**.
+     The page opens to a **Configure New Runtime** form with the Docker tab
+     selected.
 
-    You can expand the codeblock by clicking **Show** to see the entire script.
+1. Click **Copy Script**.
 
-4. Replace the placeholder for `<your-password>` with your own
+1. Replace the placeholder for `<your-password>` with your own
 {{site.konnect_saas}} password.
 
-4. Run the script on any host you choose.
+1. Run the script on any host you choose.
 
     This script creates a Docker container running a simple
     {{site.base_gateway}} instance and connects it to your
     {{site.konnect_saas}} account.
 
-5. Click **Done** to go to the Runtime Manager page.
+1. Click **Done** to go to the Runtime Manager page.
 
 Once the script has finished running, the Runtime Manager will include
-a new entry for your instance.
-
-<!-- and the tag in the **Node Status** column should say
-**Connected**.-->
+a new entry for your instance and the tag in the **Sync Status** column should
+say **Connected**.
 
 The default proxy URL for this runtime is `http://localhost:8000`. Take
 note of this URL, as you'll need it later to access a Service
 implementation.
 
-<!-- To change the default URL, see [link TBA].-->
-
-<div class="alert alert-ee warning">
-<b>Important:</b> {{site.konnect_saas}} provisions certificates for the data
+{:.important}
+> Important: {{site.konnect_saas}} provisions certificates for the data
```

Comment:
@lena-larionova Post helium, Kong is no longer going to generate the certificate server side. With client side certificates, the expiration is whatever the user sets it to when they create the certificate. In the [setup script](https://raw.githubusercontent.com/Kong/konnect-runtimes/pinned-cert/konnect-runtime-setup.sh), the openssl command is using 3650 days (10yrs).

Bottom line, we need to revisit the contents of this Renew Certificates link.

Reply:
Noted. Will have to take this on as a separate task. For now, I can remove these notes from the configuration topics.